app/src/main/java/com/example/parkingsystem/mvp/model/ParkingReservationModel.kt

```diff
@@ -0,0 +1,55 @@
+package com.example.parkingsystem.mvp.model
+
+import com.example.parkingsystem.mvp.contract.ParkingReservationContract
+import com.example.parkingsystem.util.DateUtils
+import java.text.SimpleDateFormat
+import java.util.*
```

Comment:
after the configuration you can remove this one as well

Reply:
Done